WireMessaging/Sources/WireMessagingUI/WireCells/Components/Files/FilesViewModel.swift

```diff
@@ -52,16 +52,37 @@ package final class FilesViewModel: ObservableObject {
         static let loadMoreThreshold = 5
     }
 
+    enum State {
+        case loading
+        case received(items: [FilesViewItem])
+        case noData
+        case pending // cells are not ready yet
+
+        var items: [FilesViewItem] {
+            switch self {
+            case let .received(items):
+                items
+            default:
+                []
+            }
+        }
+    }
+
     private let fetchNodesUseCase: WireCellsFetchNodesUseCase
+    private let isCellsStatePending: Bool
 
-    package init(fetchNodesUseCase: WireCellsFetchNodesUseCase) {
+    package init(
+        fetchNodesUseCase: WireCellsFetchNodesUseCase,
+        isCellsStatePending: Bool
+    ) {
         self.fetchNodesUseCase = fetchNodesUseCase
+        self.isCellsStatePending = isCellsStatePending
```

Comment:
```suggestion
        self.State = isCellsStatePending ? .pending : .loading
```

I think this works just as well and allows us to delete the `isCellsStatePending` property.

Reply:
done in https://github.com/wireapp/wire-ios/pull/3512/commits/9bd117a84545ab1d1a0d82405e94487df553b06f